docs/guide/migrationV2.md

```diff
@@ -8,3 +8,5 @@ A number of changes were made to the configuration options passed to the plugin
 
  * `xScaleID` option default has been changed, now set to `undefined`. If the option is missing, the plugin will try to use the first scale of the chart, configured as `'x'` axis. If more than one scale has been defined in the chart as `'x'` axis, the option is mandatory to select the right scale.
  * `yScaleID` option default has been changed, now set to `undefined`. If the option is missing, the plugin will try to use the first scale of the chart, configured as `'y'` axis. If more than one scale has been defined in the chart as `'y'` axis, the option is mandatory to select the right scale.
+ * When [stacked scales](https://www.chartjs.org/docs/latest/axes/cartesian/#common-options-to-all-cartesian-axes) are used and  `xMin`, `xMax`, `yMin` and `yMax` options are missing, the scale dimension is used instead of chart area to set the annotation dimension.
+ * When [stacked scales](https://www.chartjs.org/docs/latest/axes/cartesian/#common-options-to-all-cartesian-axes) are used and  `xValue` and `yValue` options are missing, the scale dimension is used instead of chart area to set the annotation location.
```

Comment:
```suggestion
* When [stacked scales](https://www.chartjs.org/docs/latest/axes/cartesian/#common-options-to-all-cartesian-axes) are used, instead of the whole chart area, the designated scale area is used as fallback for `xMin`, `xMax`, `yMin`, `yMax`, `xValue` or `yValue` options.
```

Not sure if this is any better. @etimberg what do you think?

Reply:
That makes sense to me